Applications/web_portal/web_app/library/source/pages/dashboard_page/dashboard_page.tsx

```diff
@@ -1,16 +1,88 @@
+import {css, StyleSheet} from 'aphrodite';
 import * as React from 'react';
+import {BurgerButton, HBoxLayout, Padding, VBoxLayout} from '../..';
+import {NotificationButton} from './notification_button';
 
 export interface Properties {}
 
-export interface State {}
+export interface State {
+}
 
 /** Displays the main dashboard. */
 export class DashboardPage extends React.Component<Properties, State> {
-  constructor(properties: Properties) {
-    super(properties);
+  constructor(props: Properties) {
+    super(props);
   }
 
   public render(): JSX.Element {
-    return <div>Dashboard</div>;
+    return (
+      <VBoxLayout width='100%' height='100%'>
+        <div className={css(DashboardPage.STYLE.header)}>
+          <HBoxLayout className={css(DashboardPage.STYLE.leftHeader)}
+              width='136px' height='60px'>
+            <Padding size='18px'/>
+            <VBoxLayout height='60px'>
+              <Padding/>
+              <div className={css(DashboardPage.STYLE.burgerButtonWrapper)}>
+                <BurgerButton width='20px' height='14px'
+                  color='#684BC7' highlightColor='#684BC7'/>
+              </div>
+              <Padding/>
+            </VBoxLayout>
+            <Padding size='20px'/>
+            <VBoxLayout height='60px'>
+              <Padding/>
+              <div className={css(DashboardPage.STYLE.logo)}>
+                <img src='resources/dashboard/logo.png'
+                  className={css(DashboardPage.STYLE.logo)}/>
+              </div>
+              <Padding/>
+            </VBoxLayout>
+          </HBoxLayout>
+          <VBoxLayout height='60px' width='45px'
+              className={css(DashboardPage.STYLE.rightHeader)}>
+            <Padding/>
+            <div className={css(DashboardPage.STYLE.notificationButtonWrapper)}>
+              <NotificationButton width='15px' height='20px'/>
+            </div>
+            <Padding/>
+          </VBoxLayout>
+        </div>
+        <div className={css(DashboardPage.STYLE.separator)}/>
+      </VBoxLayout>);
   }
+  private static STYLE = StyleSheet.create({
```

Comment:
This should be removed entirely once elements are layed out using the BoxLayout elements.